Bugzilla/Comment.pm

```diff
@@ -45,6 +45,7 @@ use constant DB_COLUMNS => qw(
     already_wrapped
     type
     extra_data
+    is_markdown
```

Comment:
Does BMO still keep up with POD docs? If so we can add some docs for the `is_markdown` field though it is self descriptive :)